ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/manage/signing-key/page-old.tsx

```diff
@@ -4,12 +4,12 @@
  */
 
 import { Alert } from '@inngest/components/Alert';
+import { InlineCode } from '@inngest/components/InlineCode';
```

Comment:
These are out of scope for this PR, but is the old page still used? Isn't the feature enabled for everybody already?